### PR TITLE
Commonly used objects that enable Agile Workflows

### PR DESCRIPTION
**Pull request by yamatt**
Adds:

- A sticky note typically used to represent a task

### CODE REVIEW

**Comment by lipis** [comment]
A bit too simple.. no?

**Comment by yamatt** [comment]
Simple and elegant.... no?

I was planning on expanding it as new features came to mind, but this one solves the problem I have for now.

Is there any kind of quality checks I'm not meeting?

**Comment by dwelle** [comment]
Not necessarily quality checks in this case, but in general if it's super easy to create yourself and only contains a handful of items, it's probably a better fit for a private library. But we haven't published any guidelines yet, so...

**Comment by yamatt** [comment]
To be honest it doesn't really work very well either because the text box can't be a bound width so you have to do some work with your enter key to get the text to fix the background square.

**Comment by dwelle** [comment]
Let's rename this to a "sticky note" and update the description as well and we can merge!

**Comment by yamatt** [comment]
Sounds good. I'll try and get around to it next week.

File: libraries.json [hunk 1]
```diff
@@ -1,4 +1,17 @@
 [
+  {
+    "name": "Agile Workflow",
```
Comment (ferminrp) [inline libraries.json:3]:
```suggestion
    "name": "Sticky Note",
```

File: libraries.json [hunk 2]
```diff
@@ -1,4 +1,17 @@
...
+    "description": "Commonly used objects that enable Agile Workflows.",
```
Comment (ferminrp) [inline libraries.json:4]:
```suggestion
    "description": "Simple and Elegant Sticky Note",
```

**Review by ferminrp — CHANGES_REQUESTED** [review]

**Pull request closed without merging**